### PR TITLE
Add documentation to definitions

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/definitions/lorentz_factor.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
+"""
+
```
Comment (blackyblack) [inline symplyphysics/definitions/lorentz_factor.py:2]: Empty

File: symplyphysics/definitions/thermal_de_broglie_wavelength.py [hunk 2]
```diff
@@ -1,39 +1,68 @@
...
 )
 
-# Description
-## The thermal de Broglie wavelength can be roughly described as the average de Broglie wavelength
-## of particles in an ideal gas at a specified temperature. When compared to average interparticle
-## spacing in the gas, it can be used to tell if the gas can be considered to be a classical or
-## Maxwell-Boltzmann gas, in which case the thermal wavelength must be much smaller than the average
-## interparticle spacing. Otherwise, quantum effects must be taken into account.
-
 # Definition: lambda = hbar * sqrt(2 * pi / (m * k * T))
```
Comment (blackyblack) [inline symplyphysics/definitions/thermal_de_broglie_wavelength.py:28]: Remove

File: symplyphysics/definitions/thermal_de_broglie_wavelength.py [hunk 3]
```diff
@@ -1,39 +1,68 @@
...
+Symbol:
+    :code:`lambda`
+
+Latex:
+    :math:`\lambda`
+"""
+
 particle_mass = clone_symbol(symbols.basic.mass, "particle_mass")
+"""
+:attr:`~symplyphysics.symbols.basic.mass` of particles comprising the gas.
```
Comment (blackyblack) [inline symplyphysics/definitions/thermal_de_broglie_wavelength.py:43]: Of one particle. Someone could think it is a summary mass of the particles.

File: symplyphysics/definitions/thermal_de_broglie_wavelength.py [hunk 4]
```diff
@@ -1,39 +1,68 @@
...
+
 temperature = symbols.thermodynamics.temperature
+"""
+:attr:`~symplyphysics.symbols.thermodynamics.temperature` of the gas.
+
+Symbol:
+    :code:`T`
+"""
 
 law = Eq(thermal_wavelength,
```
Comment (blackyblack) [inline symplyphysics/definitions/thermal_de_broglie_wavelength.py:57]: We use `definition` for definitions.

File: symplyphysics/definitions/thermodynamic_compressibility.py [hunk 5]
```diff
@@ -1,41 +1,69 @@
...
 
+definition = Eq(compressibility, -1 * Derivative(volume(pressure), pressure) / volume(pressure))
+r"""
+:code:`beta = -1 / V * (dV/dp)`
 
-def print_law() -> str:
-    return print_expression(definition)
+Latex:
+    .. math::
+        \beta = - \frac{1}{V} \left( \frac{\partial V}{\partial p} \right)
```
Comment (blackyblack) [inline symplyphysics/definitions/thermodynamic_compressibility.py:65]: I think `dV/dp` braces are not needed.

File: symplyphysics/definitions/vector/angular_momentum_is_position_cross_linear_momentum.py [hunk 6]
```diff
@@ -1,3 +1,16 @@
+r"""
+Angular momentum is position cross linear momentum
+==================================================
+
+The pseudovector of *angular momentum* of a particle is defined relative to a fixed point,
+usually an origin, as a cross product of its position vector and linear momentum.
```
Comment (blackyblack) [inline symplyphysics/definitions/vector/angular_momentum_is_position_cross_linear_momentum.py:6]: Hm, I think vectors are always defined relative to an origin.

File: symplyphysics/definitions/vector/angular_momentum_is_position_cross_linear_momentum.py [hunk 7]
```diff
@@ -7,25 +20,60 @@
...
+        .. math::
+            \vec L = \vec r \times \vec p
+
+    :param position_vector\_: position vector of the particle relative to a fixed point.
+
+        Symbol: :code:`r`
+
+        Latex: :math:`\vec r`
 
+        Dimension: :math:`\mathsf{L}`
```
Comment (blackyblack) [inline symplyphysics/definitions/vector/angular_momentum_is_position_cross_linear_momentum.py:44]: Simply write `length` here. We do not use dimensional analysis.

File: symplyphysics/definitions/vector/angular_momentum_is_position_cross_linear_momentum.py [hunk 8]
```diff
@@ -7,25 +20,60 @@
...
+
+    :param linear_momentum\_: pector of linear momentum of the particle.
```
Comment (blackyblack) [inline symplyphysics/definitions/vector/angular_momentum_is_position_cross_linear_momentum.py:46]:
```suggestion
    :param linear_momentum\_: vector of linear momentum of the particle.
```

File: symplyphysics/definitions/vector/angular_momentum_is_position_cross_linear_momentum.py [hunk 9]
```diff
@@ -7,25 +20,60 @@
...
+
+        Symbol: :code:`p`
+
+        Latex: :math:`\vec p`
+
+        Dimension: :math:`\mathsf{M} \mathsf{L} \mathsf{T}^{-1}`
```
Comment (blackyblack) [inline symplyphysics/definitions/vector/angular_momentum_is_position_cross_linear_momentum.py:52]: `momentum` here

File: symplyphysics/definitions/vector/damping_force_is_proportional_to_velocity.py [hunk 10]
```diff
@@ -9,24 +18,76 @@
...
 
-# Law: F = -b * v
-## F - vector of damping force
-## b - damping constant, b >= 0
-## v - vector of object's velocity
+Symbol:
+    :code:`b`
 
-damping_constant = Symbol("damping_constant", units.mass / units.time)
+Dimension: :math:`\mathsf{M} \mathsf{T}^{-1}`
```
Comment (blackyblack) [inline symplyphysics/definitions/vector/damping_force_is_proportional_to_velocity.py:28]: Remove dimension here to be consistent. We will likely introduce some auto generation of dimensions later.

File: symplyphysics/definitions/volumetric_coefficient_of_thermal_expansion.py [hunk 11]
```diff
@@ -1,41 +1,67 @@
...
 )
+r"""
+:code:`alpha_V = 1 / V * (dV/dT)_p`
 
-
-def print_law() -> str:
-    return print_expression(definition)
+Lambda:
+    .. math::
+        \alpha_V = \frac{1}{V} \left( \frac{d V}{d T} \right)_p
```
Comment (blackyblack) [inline symplyphysics/definitions/volumetric_coefficient_of_thermal_expansion.py:63]: Useless braces.

File: symplyphysics/core/vectors/arithmetics.py [hunk 12]
```diff
@@ -44,7 +44,7 @@ def equal_vectors(vector_left: Vector, vector_right: Vector) -> bool:
 def add_cartesian_vectors(*vectors: Vector) -> Vector:
 
     def add_two_cartesian_vectors(vector_left: Vector, vector_right: Vector) -> Vector:
-        if vector_left.coordinate_system != vector_right.coordinate_system:
+        if vector_left.coordinate_system is not vector_right.coordinate_system:
```
[the remark this answers is not in the document]
Comment (alesanter) [reply symplyphysics/core/vectors/arithmetics.py:47]: I accidentally left it after investigating why the tests fail. I'll revert it.

File: symplyphysics/definitions/volumetric_coefficient_of_thermal_expansion.py (hunk 11 above)
Comment (alesanter) [reply symplyphysics/definitions/volumetric_coefficient_of_thermal_expansion.py:63]: There's the `_p` index after them so they're needed.

File: symplyphysics/definitions/vector/angular_momentum_is_position_cross_linear_momentum.py (hunk 6 above)
Comment (alesanter) [reply symplyphysics/definitions/vector/angular_momentum_is_position_cross_linear_momentum.py:6]: Not really. The vector of linear momentum, for example, is the same, no matter what origin we choose. I'll reword the description though.

Comment (alesanter) [reply symplyphysics/definitions/vector/angular_momentum_is_position_cross_linear_momentum.py:6]: I'm guessing you mean the _representation_ of a vector changes depending on the spacial orientation and origin position of the coordinate system, but it's about vector as an entity independent of the coordinate system.

File: symplyphysics/definitions/acceleration_is_velocity_derivative.py [hunk 13]
```diff
@@ -18,15 +18,15 @@
 :attr:`~symplyphysics.symbols.kinematic.acceleration` of the body as a function of time.
 
 Symbol:
-    :code:`a`
+    :code:`a = a(t)`
```
Comment (blackyblack) [inline symplyphysics/definitions/acceleration_is_velocity_derivative.py:21]: Why not simply `a(t)`?

**Review by blackyblack — APPROVED** [review]

File: symplyphysics/definitions/acceleration_is_velocity_derivative.py (hunk 13 above)
Comment (alesanter) [reply symplyphysics/definitions/acceleration_is_velocity_derivative.py:21]: I thought that in that case I would need to make the law look something like `a(t) = d(v(t))/dt` for consistency, where function application seems redundant. Can leave those simply as `a(t)` of course.

Comment (blackyblack) [reply symplyphysics/definitions/acceleration_is_velocity_derivative.py:21]: Yes, for consistency they should be updated in laws as well.

File: symplyphysics/definitions/harmonic_oscillator_is_second_derivative_equation.py [hunk 14]
```diff
@@ -1,32 +1,59 @@
...
+"""
 
 definition = Eq(Derivative(displacement_function(time), (time, 2)),
     -angular_frequency**2 * displacement_function(time))
+r"""
+:code:`Derivative(x(t), (t, 2)) = -1 * w^2 * x(t)`
+
+Latex:
+    .. math::
+        \frac{d^2 x(t)}{d t^2} = - \omega^2 x(t)
```
Comment (blackyblack) [inline symplyphysics/definitions/harmonic_oscillator_is_second_derivative_equation.py:55]:
```suggestion
        \frac{d^2 x}{d t^2} = - \omega^2 x(t)
```

**Review by blackyblack — APPROVED** [review]

**Pull request merged**